source/agora/test/Base.d

```diff
@@ -2349,6 +2349,11 @@ public APIManager makeTestNetwork (APIManager : TestAPIManager = TestAPIManager)
       event_handlers : test_conf.event_handlers,
       registry: {
           enabled: true,
+          realm: {
+              authoritative: SetInfo!bool(true, true),
+              primary: SetInfo!string("name.registry", true),
```

Comment:
Shouldnt this be an IP as well?

Reply:
No, this is only used for NS record right now